### DISCUSSION TITLE
Likelihood of an official s390x release?

### DISCUSSION THREAD

**Discussion by aroberts87** [mozilla/sccache#1901] [General]
Hey all! 👋 I was wondering if there was an appetite for an official s390x release, or what the impediment is if there isn't one. Quickly searching the Internet for "sccache s390x" shows a couple hits of people building locally, and at this moment I am also looking into this as I need x86_64, aarch64, and s390x binaries. Thoughts?

**Comment by sylvestre** [comment]
Hello
See https://github.com/mozilla/sccache/issues/1442
Which was fixed last week (i didn't know)
So, the blocker is removed!

**Comment by aroberts87** [comment]
@sylvestre, that's fantastic news! I had seen that issue, but noticed it was so old I wasn't sure if there was a change in status or not. Now that the blocker has been removed do you foresee a release with s390x in the near future? If so, will it be `musl` like the other Linux options?

**Reply by sylvestre** [reply]
ring isn't a direct dependency
https://github.com/Keats/jsonwebtoken/pull/333 needs to land before we can test s390x support

**Reply by aroberts87** [reply]
Ah, gotcha. Thanks for clarifying!
